src/main.c

```diff
@@ -495,7 +496,7 @@ static gboolean resign_start(int argc, char **argv)
 		goto out;
 	}
 
-	if (!check_bundle(argv[2], &bundle, !verification_disabled, &ierror)) {
+	if (!check_bundle(argv[2], &bundle, !verification_disabled, FALSE, &ierror)) {
```

Comment:
For parameters that are essential for security, I find this hard to read (especially `, TRUE, FALSE,` or `, FALSE, FALSE` below). How about using a flags enum for this? `R_CHECK_DISABLE_VERIFICATION` and `R_CHECK_TRUST_ENVIRONMENT` for now?

I thought about creating the bundle struct outside of `check_bundle` with parameters, but I quite like that you need to call `check_bundle` to get a bundle struct in the current code.

Reply:
+1 for struct from my side. I actually already thought about this too, as I always had to check what 'FALSE' and 'TRUE' actually meant.

Reply:
"+1 for Enum" I meant of course. :smirk:

Reply:
@ejoerns What do you think about building this on https://github.com/rauc/rauc/pull/462 instead?

Reply:
I've rebased this on after the merge of #462, which simplified this significantly.